spec/models/abuse_report_spec.rb

```diff
@@ -345,4 +345,22 @@
       expect(safe_report.save).to be_truthy
     end
   end
+
+  describe "#attach_work_download" do
+    include ActiveJob::TestHelper
+
+    let(:work) { create(:work) }
```

Comment:
It looks like this wouldn't be created for the first test; is that intended?

Reply:
Yes, it is.